### PR TITLE
updated to pipestat 0.6.0

### PR DESCRIPTION
**Pull request by khoroshevskyi**

### CODE REVIEW

File: pypiper/manager.py [hunk 1]
```diff
@@ -437,7 +436,7 @@ def _completed(self):
         :return bool: Whether the managed pipeline is in a completed state.
         """
         return (
-            self.pipestat.get_status(self._pipestat_manager.sample_name)
+            self.pipestat.get_status(self._pipestat_manager.cfg["record_identifier"])
```
Comment (donaldcampbelljr) [inline pypiper/manager.py:439]: should be able to use `self._pipestat_manager.record_identifier` now that the properties are fixed.

File: pypiper/manager.py [hunk 2]
```diff
@@ -771,7 +774,7 @@ def _set_status_flag(self, status):
         prev_status = self.status
         self.status = status
         self.pipestat.set_status(
-            record_identifier=self._pipestat_manager.sample_name,
+            record_identifier=self._pipestat_manager.cfg["record_identifier"],
```
Comment (donaldcampbelljr) [inline pypiper/manager.py:777]: see above, can use `self._pipestat_manager.record_identifier`

File: pypiper/manager.py [hunk 3]
```diff
@@ -788,7 +791,7 @@ def _flag_file_path(self, status=None):
         """
 
         flag_file_name = "{}_{}_{}".format(
-            self._pipestat_manager["_pipeline_name"],
+            self._pipestat_manager.cfg["pipeline_name"],
```
Comment (donaldcampbelljr) [inline pypiper/manager.py:794]: can use `self._pipestat_manager.pipeline_name`

File: pypiper/utils.py [hunk 4]
```diff
@@ -1113,6 +1113,8 @@ def _add_args(parser, args, required):
 def result_formatter_markdown(pipeline_name, record_identifier, res_id, value) -> str:
     """
     Returns Markdown formatted value as string
+
+    # Pipeline_name and record_identifier should be kept because pipestat needs it
```
Comment (donaldcampbelljr) [inline pypiper/utils.py:1117]: Can the input be changed to be more general **kwargs?

**Review by donaldcampbelljr — APPROVED** [review]
Minor things but non are necessary

File: pypiper/manager.py (hunk 2 above)
Comment (nsheff) [reply pypiper/manager.py:777]: we should discourage using private variables outside of the package though -- is there another way to do this?

if this is really necessary, it indicates pipestat needs some other way to do this.

Comment (donaldcampbelljr) [reply pypiper/manager.py:777]: It looks like this is simply that pypiper sets its pipestatmanager attribute as private:
https://github.com/databio/pypiper/blob/1c2c844dacf29c1d693af20e57f20e9cc1b60d4d/pypiper/manager.py#L345-L358

While the `record_identifier` attribute on the pipestatmanager is _not_ private.

Should we just change this to be `self.pipestat_manager = PipestatManger()` ?

Comment (nsheff) [reply pypiper/manager.py:777]: Oh I think I must have just misinterpreted things. I thought it was accessing a private variable from pipestat. If it's just accessing its own private variables, that's fine !

**Pull request merged**